docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb

```diff
@@ -0,0 +1,411 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1bf4de00",
+   "metadata": {},
+   "source": [
+    "# Faster Prediction with TensorRT\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/autogluon/autogluon/blob/master/docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb)\n",
+    "[![Open In SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/autogluon/autogluon/blob/master/docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49712a2a",
+   "metadata": {},
+   "source": [
+    "[TensorRT](https://developer.nvidia.com/tensorrt), built on the NVIDIA CUDA® parallel programming model, enables us to optimize inference by leveraging libraries, development tools, and technologies in NVIDIA AI, autonomous machines, high-performance computing, and graphics. AutoGluon-MultiModal is now integrated with TensorRT via `predictor.optimize_for_inference()` interface. This tutorial demonstates how to leverage TensorRT in boosting inference speed, which would be helpful in increasing efficiency at deployment environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2c57755d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import time\n",
+    "import warnings\n",
+    "from IPython.display import clear_output\n",
+    "warnings.filterwarnings('ignore')\n",
+    "np.random.seed(123)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f4b5836",
+   "metadata": {},
+   "source": [
+    "### Install tensorrt package\n",
+    "Since the tensorrt package is currently an optional dependency of autogluon.multimodal, we need to ensure tensorrt package is correctly installed."
```

Comment:
Do users also need to install `onnx` or `onnxruntime-gpu` to use TensorRT? They seem not in our installation by default.

Reply:
Installing onnxruntime-gpu and onnx package, along with tensorrt package via
```bash
!pip install autogluon.multimodal[tests]
```